### PR TITLE
deps: upgrade x/tools and gopls to 0cba7a3a

### PR DESCRIPTION
**Pull request by myitcv**

### CODE REVIEW

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt [hunk 1]
```diff
@@ -10,14 +10,12 @@ errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagn
...
 vim ex 'e main.go'
-errlogmatch -peek 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
 vim ex 'e other.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
-errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
-errlogmatch -count=2 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
+errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
```
Comment (leitzler) [inline cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt]: Is this really how things should work? (I notice that the tests passed so I assume it matches the gopls behaviour here). Aren't we supposed to get diagnostics when we open a file, even if it's just empty?

If the code is indeed correct, we need to update the test description at the top of this file.

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_compile_errors_in_one.txt [hunk 2]
```diff
@@ -9,13 +9,11 @@ errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagn
...
 
 vim ex 'e main.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 
 vim ex 'e other.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
-errlogmatch -count=2 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
-errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
+errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
```
Comment (leitzler) [inline cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_compile_errors_in_one.txt]: Same as previous comment, this test need it's description updated.

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_no_errors.txt [hunk 3]
```diff
@@ -8,13 +8,11 @@ sleep $GOVIM_ERRLOGMATCH_WAIT
...
 
 vim ex 'e main.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
 
 vim ex 'e other.go'
-errlogmatch 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
 
-errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
-errlogmatch -count=1 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/other.go
+errlogmatch -count=0 -start 'PublishDiagnostics callback: &protocol.PublishDiagnosticsParams{\n\S+:\s+URI:\s+"file://'$WORK/main.go
```
Comment (leitzler) [inline cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_no_errors.txt]: Test description needs to be updated

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt (hunk 1 above)
Comment (myitcv) [reply cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt]: > Aren't we supposed to get diagnostics when we open a file, even if it's just empty?

Not any more. So yes, this is now correct.

Comment (myitcv) [reply cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_analysis_errors_in_one.txt]: Updated the description in this and other files. Thanks for that catch.

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_compile_errors_in_one.txt (hunk 2 above)
Comment (myitcv) [reply cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_compile_errors_in_one.txt]: Done

File: cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_no_errors.txt (hunk 3 above)
Comment (myitcv) [reply cmd/govim/testdata/scenario_diagnostic_assertions/multiple_files_no_errors.txt]: Done.

**Pull request merged**